### DISCUSSION TITLE
Is it a bug？

### DISCUSSION THREAD

**Discussion by chenyunchi** [cpeditor/cpeditor#1414] [Q&A]
![无标题](https://github.com/user-attachments/assets/d3ce1e1a-d808-4f15-9da4-52bb05a67138)
As you can see, the second line and the following are the same, but it shows that my answer is wrong.
In normal times, it will also report an error at the end of each line.

**Comment by ouuan** [comment]
Please copy-paste the content of the output and answer as text. Try to minimize it (e.g. only the first few lines).
